consensus/cosipbft/mod.go

```diff
@@ -92,14 +93,16 @@ func (c *Consensus) Listen(v consensus.Validator) (consensus.Actor, error) {
 }
 
 type pbftActor struct {
+	closing     chan struct{}
 	hashFactory crypto.HashFactory
 	cosiActor   cosi.Actor
 	rpc         mino.RPC
 }
 
-// Propose takes the proposal and send it to the participants of the consensus.
-// It returns nil if the consensus is reached and that the participant are
-// committed to it, otherwise it returns the refusal reason.
+// Propose implements consensus.Actor. It takes the proposal and send it to the
+// participants of the consensus. It returns nil if the consensus is reached and
+// that the participant are committed to it, otherwise it returns the refusal
+// reason.
```

Comment:
```suggestion
// Propose implements consensus.Actor. It takes the proposal and sends it to the
// participants of the consensus. It returns nil if the consensus is reached and
// the participant are committed to it, otherwise it returns the refusal reason.
```